core/src/integration-test/java/com/scalar/db/storage/jdbc/JdbcAdminImportTestUtils.java

```diff
@@ -179,9 +187,14 @@ private LinkedHashMap<String, String> prepareColumnsForMysql() {
     columns.put("col18", "TINYBLOB");
     columns.put("col19", "MEDIUMBLOB");
     columns.put("col20", "LONGBLOB");
-    columns.put("col21", "BINARY(255)");
+    columns.put("col21", "BINARY(5)");
```

Comment:
I changed the column size to facilitate inserting a BLOB value for the simple write-read integration test. For this data type, the value is right-padded if the value length is inferior to the column size. 
I did a similar change for some other data types that are right padded.